ext/EnzymeExt.jl

```diff
@@ -0,0 +1,12 @@
+module EnzymeExt
+
+using MeshIntegrals
+using Meshes: Geometry
+using Enzyme
```

Comment:
```suggestion
using Enzyme: Enzyme
```
We generally prefer explicit imports.